virttest/qemu_monitor.py

```diff
@@ -1835,19 +1835,22 @@ def migrate_set_downtime(self, value):
         args = {"value": val}
         return self.cmd("migrate_set_downtime", args)
 
-    def live_snapshot(self, device, snapshot_file, snapshot_format="qcow2"):
+    def live_snapshot(self, device, snapshot_file, snapshot_format="qcow2",
+                      snapshot_mode="absolute-paths"):
```

Comment:
Oh, you are right. qmp-commands.hx:          (NewImageMode, optional, default 'absolute-paths')